src/main/resources/application.properties

```diff
@@ -1,17 +1,18 @@
 #Server properties
-server.port=9004
+server.port=8080
 #Datasource properties
-spring.datasource.driverClassName=com.mysql.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/login_db
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/login_db?createDatabaseIfNotExist=true&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=Asia/Jakarta&useSSL=false&allowPublicKeyRetrieval=true
 spring.datasource.username=root
-spring.datasource.password=pass
+spring.datasource.password=root
 spring.datasource.testWhileIdle=true
 spring.datasource.validationQuery=SELECT 1
 #JPA properties. Using ddl-auto = create will drop schema every-time.
 #Choose the correct property based on development / production role.
-spring.jpa.hibernate.ddl-auto=create
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
-spring.datasource.initialization-mode=always
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.generate-ddl=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+#spring.sql.init.mode=always
```

Comment:
This breaks the instructions in README. Only the swagger changes make sense here.

Reply:
Is it possible to update the README?

Reply:
Yes it is,

Reply:
I will update the README.md file then.

Reply:
Kindly do not update `spring.jpa.hibernate.ddl-auto=create` to `update`. This will not create the schema when users first start the app. 

Do not change the `dialect/init-mode` etc.

The scope of your PR is limited to Spring Boot upgrade and Swagger changes. Kindly do not piggy back unrelated changes in the same PR without reason. Feel free to create another PR for these side changes.

Reply:
OK.